src/app/accounts/settings.component.ts

```diff
@@ -74,10 +75,11 @@ export class SettingsComponent implements OnInit {
     }
 
     async ngOnInit() {
-        this.showMinToTray = this.platformUtilsService.getDevice() === DeviceType.WindowsDesktop;
+        this.showMinToTray = this.platformUtilsService.getDevice() === DeviceType.WindowsDesktop || this.platformUtilsService.getDevice() === DeviceType.LinuxDesktop;
```

Comment:
We sure this works well on Linux? Seems that it was explicitly set to windows for a reason.

Reply:
I have only tested this on Archlinux where it works fine

Reply:
Just tested it under ubuntu 18.10, seems to have a bug where it duplicates the tray icon each time the window gets minimized/closed. I have to investigate this.

Reply:
@kspearrin I have now reverted the change. So the tray functionality is only available on Windows.